code/game/objects/items/ego_weapons/non_abnormality/wcorp.dm

```diff
@@ -0,0 +1,166 @@
+//Like this so we can add a charge mechanic to one of them and have it carry down.
+/obj/item/ego_weapon/city/wcorp
+	name = "W corp baton"
+	desc = "A glowing blue baton used by W corp employees."
+	icon_state = "wbatong"
+	inhand_icon_state = "wbatong"
+	force = 18
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("bashes", "crushes")
+	attack_verb_simple = list("bash", "crush")
+	var/release_message = "You release your charge, damaging your opponent!"
+	var/charge_effect = "deal an extra attack in damage."
+	var/charge_cost = 2
+	var/charge
+	var/activated
+
+/obj/item/ego_weapon/city/wcorp/attack_self(mob/user)
+	..()
+	if(charge>=charge_cost)
+		to_chat(user, "<span class='notice'>You prepare to release your charge.</span>")
+		activated = TRUE
+	else
+		to_chat(user, "<span class='notice'>You don't have enough charge.</span>")
+
+/obj/item/ego_weapon/city/wcorp/examine(mob/user)
+	. = ..()
+	. += "Spend [charge]/[charge_cost] charge to [charge_effect]"
```

Comment:
```suggestion
	. += "Spend [charge_cost]/[charge] charge to [charge_effect]"
```
As it reads right now, it's "Spend X out of Y charge to Z"
Please either implement the above change or simply switch it to be a little wordier.